docs/LwM2M_Client/Nordic/nRF9160.md

```diff
@@ -6,7 +6,7 @@ Integrate your Nordic nRF9160 DK board.
 
 - The nRF9160 DK board with a USB cable.
 - Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
-- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download).
+- {{ nRF9160DK_flashing_tool_prerequisit }}
```

Comment:
Can someone tell me why exactly we are doing this? This macro is useless as it's being used in one place only???

Reply:
The goal is to have one document with text for all zephyrs board and only change macros in places where guide for these boards differ. Finally this is going to be automated, but this is going to be done in separate task.